stable/dex-controller/values.yaml

```diff
@@ -9,7 +9,7 @@ controller:
     resources:
       limits:
         cpu: 100m
-        memory: 100Mi
+        memory: 250Mi
```

Comment:
During the MWT, it was stable after going to 750Mi.

At 300Mi it was still being OOMed.

Reply:
Yeah, let's set to 512Mi